packages/office-ui-fabric-react/src/components/Pivot/Pivot.styles.ts

```diff
@@ -17,19 +114,107 @@ export const getStyles = (props: IPivotStyleProps): IPivotStyles => {
         color: palette.themePrimary,
         whiteSpace: 'nowrap'
       },
+      rootIsLarge && [classNames.rootIsLarge],
```

Comment:
no need to wrap in array.

Reply:
Yeah this evolved from something more complicated and I didn't unarray it